src/renderer/shared/core/types/track.ts

```diff
@@ -0,0 +1,15 @@
+import { BN } from '@polkadot/util';
+
+import type { BlockHeight } from './general';
+
+export type TrackId = string;
+
+export type TrackInfo = {
+  name: string;
+  maxDeciding: BN;
+  decisionDeposit: BN;
+  preparePeriod: BlockHeight;
+  decisionPeriod: BlockHeight;
+  // minApproval?: VotingCurve?;
+  // minSupport?: VotingCurve;
```

Comment:
Do you need this commented code? Maybe uncomment it because it's optional it will not affect on this type